src/evaluators/CallExpression.js

```diff
@@ -121,7 +121,9 @@ function generateRuntimeCall(
     let argStart = 1;
     if (thisArg instanceof Value) {
       if (typeof propName === "string") {
-        callFunc = t.memberExpression(nodes[0], t.identifier(propName), !t.isValidIdentifier(propName));
+        let isValidIdentifier = t.isValidIdentifier(propName);
+        let property = isValidIdentifier ? t.identifier(propName) : t.stringLiteral(propName);
+        callFunc = t.memberExpression(nodes[0], property, !isValidIdentifier);
```

Comment:
In terms of code clarity, what I suggested is:

```js
callFunc = t.isValidIdentifier(propName) ?
  t.memberExpression(nodes[0], t.identifier(propName), false) :
  t.memberExpression(nodes[0], t.stringLiteral(propName), true);
```

I think this makes it more obvious why both branches exist.

I don't feel strongly about this though.